scripts/auto-stop-idle/autostop.py

```diff
@@ -82,10 +82,13 @@ def get_notebook_name():
         _logs = json.load(logs)
     return _logs['ResourceName']
 
+# This is hitting Jupyter's sessions API: https://github.com/jupyter/jupyter/wiki/Jupyter-Notebook-Server-API#Sessions-API
```

Comment:
Just FYI, there is nicely rendered page for these APIs available as well
http://petstore.swagger.io/?url=https://raw.githubusercontent.com/jupyter/notebook/master/notebook/services/api/api.yaml#/sessions